pkgs/top-level/python-packages.nix

```diff
@@ -8131,6 +8176,30 @@ in modules // {
     };
   };
 
+  django_raster = buildPythonPackage rec {
+    name = "django-raster-${version}";
+    version = "0.1.7a1";
+
+    # Test data is missing
+    doCheck = false;
+
+    src = pkgs.fetchFromGitHub {
+      sha256 = "1ssvs3yf3wsk0yjsbzg9bla6kfr8dqv3hxbm2g4f3bfkxp4kj5gq";
+      rev = "011cd72d1308164b133db72123cdd2cd86fd9ed6";
+      repo = "django-raster";
+      owner = "geodesign";
+    };
+
+    buildInputs = with self ; [ django_1_9 ];
+    propagatedBuildInputs = with self ; [ numpy django_colorful pillow psycopg2 pyparsing celery
+                                          (pkgs.gdal.override {pythonPackages = self;}) ];
```

Comment:
and if django is already added correctly, then there is no need to override `gdal` again, since the overridden version already comes with django